packages/grpc-js/src/server-call.ts

```diff
@@ -107,6 +108,19 @@ export class ServerReadableStreamImpl<RequestType, ResponseType> extends
       private _deserialize: Deserialize<RequestType>) {
     super({objectMode: true});
     this.cancelled = false;
+    this.call.setupReadable(this);
+  }
+
+  _read(size: number) {
+    this.call.resume();
+  }
+
+  deserialize(input: Buffer): RequestType|null {
+    if (input === null || input === undefined) {
```

Comment:
The function type declaration says this will never be null or undefined. And this is intended to only be called by our own internal code. And the code that calls it explicitly checks for null first. So this whole condition should be unnecessary.

Reply:
OK. I've removed it.